src/sonic-yang-mgmt/tests/libyang-python-tests/test_sonic_yang.py

```diff
@@ -46,7 +69,7 @@ def load_yang_model_file(self, yang_s, yang_dir, yang_file, module_name):
             raise
 
     #test load and get yang module
-    def test_load_yang_model_files(self, data, yang_s):
+    def blocked_test_load_yang_model_files(self, data, yang_s):
```

Comment:
I will commit changes for sample yang files,  after my commit, please change the test functions back.  Similar for other blocked tests.

Reply:
Sure, I approved that change. I will rebase this and remove blocked. At any stage, we should make sure, build goes fine.